src/globus_sdk/services/auth/client/service_client.py

```diff
@@ -631,3 +681,1111 @@ def delete_project(self, project_id: UUIDLike) -> GlobusHTTPResponse:
+
+                .. code-block:: pycon
+
+                    >>> ac = globus_sdk.AuthClient(...)
+                    >>> r = ac.get_clients()
+                    >>> r.data
+                    {
+                      'clients': [
+                        {
+                          'required_idp': None,
+                          'name': 'Great client of FOO',
+                          'redirect_uris': [],
+                          'links': {'privacy_policy': None, 'terms_and_conditions': None},
+                          'scopes': [],
+                          'grant_types': ['authorization_code', 'client_credentials', 'refresh_token'],
+                          'id': 'b6001d11-8765-49d3-a503-ba323fc74eee',
+                          'prompt_for_named_grant': False,
+                          'fqdns': ['foo.net'],
+                          'project': 'da84e531-1afb-43cb-8c87-135ab580516a',
+                          'client_type': 'client_identity',
+                          'visibility': 'private',
+                          'parent_client': None,
+                          'userinfo_from_effective_identity': True,
+                          'preselect_idp': None,
+                          'public_client': False,
+                        },
+                        {
+                          'required_idp': None,
+                          'name': 'Lessor client of BAR',
+                          'redirect_uris': [],
+                          'links': {'privacy_policy': None, 'terms_and_conditions': None},
+                          'scopes': [],
+                          'grant_types': ['authorization_code', 'client_credentials', 'refresh_token'],
+                          'id': 'b87f7415-ddf9-4868-8e55-d10c065f733d',
+                          'prompt_for_named_grant': False,
+                          'fqdns': ['bar.org'],
+                          'project': 'da84e531-1afb-43cb-8c87-135ab580516a',
+                          'client_type': 'client_identity',
+                          'visibility': 'private',
+                          'parent_client': None,
+                          'userinfo_from_effective_identity': True,
+                          'preselect_idp': None,
+                          'public_client': False,
+                        }
+                      ]
+                    }
+
+            .. tab-item:: Example Response Data
+
+                .. expandtestfixture:: auth.get_clients
+
+            .. tab-item:: API Info
+
+                ``GET /v2/api/clients``
+
+                .. extdoclink:: Get Clients
+                    :ref: auth/reference/#get_clients
+        """  # noqa: E501
+        return GetClientsResponse(self.get("/v2/api/clients"))
+
+    def create_client(
+        self,
+        name: str,
+        public_client: bool,
+        project: UUIDLike,
+        visibility: t.Literal["public", "private"] | utils.MissingType = utils.MISSING,
+        *,
```

Comment:
Similar to the other client class, I think this is better:
```suggestion
        *,
        visibility: t.Literal["public", "private"] | utils.MissingType = utils.MISSING,
```
but let me know if there's a reason to keep it as a positional-or-keyword arg.

Reply:
fixed in upcoming commit